src/libraries/System.Private.CoreLib/src/System/IO/LegacyFileStreamStrategy.Unix.cs

```diff
@@ -703,11 +622,6 @@ private ValueTask WriteAsyncInternal(ReadOnlyMemory<byte> source, CancellationTo
             }, this, CancellationToken.None, TaskContinuationOptions.DenyChildAttach, TaskScheduler.Default));
         }
 
-        public override Task CopyToAsync(Stream destination, int bufferSize, CancellationToken cancellationToken) =>
-            // Windows version overrides this method, so the Unix version does as well, but it doesn't
-            // currently have any special optimizations to be done and so just calls to the base.
-            base.CopyToAsync(destination, bufferSize, cancellationToken);
```

Comment:
I am not 100% sure, but I think that we wanted to have Windows and Unix `FileStream` implementations `override` the same methods. So in case, someone uses reflection to get a list of overridden methods the results are the same on Windows and Unix? I am just guessing here.

With my change it can be removed as the public type (`FileStream`) does that and the internal types (`LegacyFileStreamStrategy`) are never exposed outside.